test/core/xds/xds_cluster_resource_type_test.cc

```diff
@@ -109,8 +110,10 @@ class XdsClusterTest : public ::testing::Test {
               bootstrap.status().ToString().c_str());
       GPR_ASSERT(false);
     }
-    return MakeRefCounted<XdsClient>(std::move(*bootstrap),
-                                     /*transport_factory=*/nullptr);
+    return MakeRefCounted<XdsClient>(
+        std::move(*bootstrap),
+        /*transport_factory=*/nullptr,
+        grpc_event_engine::experimental::GetDefaultEventEngine());
```

Comment:
You can just pass nullptr here, since this test does not actually use the EE.

Reply:
Fixed.